packages/browser-agent-core/features/page-view-timing/instrument/index.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { handle } from '../../../common/event-emitter/handle'
+import { subscribeToVisibilityChange } from '../../../common/window/visibility'
+import { eventListenerOpts } from '../../../common/event-listener/event-listener-opts'
+import { gosNREUM } from '../../../common/window/nreum'
+import { getOffset, now } from '../../../common/timing/now'
+import { getConfigurationValue } from '../../../common/config/config'
+import { FeatureBase } from '../../../common/util/feature-base'
+
+export class Instrument extends FeatureBase {
+  constructor(agentIdentifier) {
+    super(agentIdentifier)
+    this.pageHiddenTime
+    this.performanceObserver
+    this.lcpPerformanceObserver
+    this.clsPerformanceObserver
+    this.fiRecorded = false
+
+
+    // // NREUM.debug("initialize page-view-timing instrument!", agentIdentifier)
+
+    if (this.isEnabled()) {
```

Comment:
This should be handled by getEnabledFeatures in the loader files:
https://github.com/newrelic/newrelic-browser-agent/blob/7922cefe29918f7a608e6f7c613b8d1c28de897f/cdn/agent-loader/lite.js#L23-L26

Reply:
agreed, can be removed